code/modules/mob/living/silicon/robot/robot_modules/station.dm

```diff
@@ -54,6 +54,7 @@ var/global/list/robot_modules = list(
 	vr_add_sprites() //Vorestation Edit: For vorestation only sprites
 
 	R.set_module_sprites(sprites)
+	sleep(50) //VOREStation edit: sprite animation
```

Comment:
I'm cautious about any sort of sleep in a `New()` proc.   Are 100% sure this object will never be present somewhere mapped in? Is it only created at runtime?

Reply:
Yes, robot_module existing on the map is a mistake of a mapper, as it is only supposed to exist as part of borg and be tightly tied to it. Otherwise it is only created upon module choice or, for borgs with pre-installed modules, upon intialization.

Reply:
Okay, apparently I fixed the thing that required that sleep(), but simply didnt remove it. Fixed now.